browser/ui/views/brave_help_bubble/brave_help_bubble_host_view.cc

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/views/brave_help_bubble/brave_help_bubble_host_view.h"
+
+#include "cc/paint/paint_shader.h"
+#include "chrome/browser/ui/views/frame/browser_view.h"
+#include "third_party/skia/include/core/SkColor.h"
+#include "third_party/skia/include/core/SkPoint.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/compositor/layer.h"
+#include "ui/gfx/canvas.h"
+#include "ui/gfx/geometry/transform_util.h"
+#include "ui/views/animation/animation_builder.h"
+#include "ui/views/background.h"
+#include "ui/views/bubble/bubble_border.h"
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
+#include "ui/views/layout/box_layout.h"
+#include "ui/views/layout/fill_layout.h"
+#include "ui/views/view_class_properties.h"
+
+namespace {
+constexpr int kWidth = 60;
+constexpr int kHeight = 60;
+constexpr base::TimeDelta kPulsingDuration = base::Milliseconds(1000);
+constexpr SkPoint kPts[] = {{0, 0}, {60, 60}};
+constexpr SkColor4f kColors[] = {{0.66, 0.10, 0.47, 1.f},
+                                 {0.22, 0.17, 0.81, 1.f}};
+constexpr SkScalar kPositions[] = {0.0, 0.65, 1.0};
+sk_sp<cc::PaintShader> kBraveGradient =
+    cc::PaintShader::MakeLinearGradient(kPts,
+                                        kColors,
+                                        kPositions,
+                                        2,
```

Comment:
You might be able to replace this hard coded value to `std::size(kPts)`

Reply:
done